.github/workflows/release.yaml

```diff
@@ -0,0 +1,146 @@
+          seperator: '-'
+  perform-release:
+    name: Perform Release
+    runs-on: ubuntu-20.04
+    needs: get-version-information
+    outputs:
+      version: ${{ needs.get-version-information.outputs.version }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Create release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          LITANI_VERSION: ${{ needs.get-version-information.outputs.version }}
+        with:
+          tag_name: litani-${{ env.LITANI_VERSION }}
+          release_name: litani-${{ env.LITANI_VERSION }}
+          body: |
+            This is LITANI version ${{ env.LITANI_VERSION }}.
+
+            ## MacOS
+
+            On MacOS, install Litani using [Homebrew](https://brew.sh/) with
+
+            ```sh
+            brew tap aws/tap
+            brew install litani
+            ```
+
+            or upgrade (if it's already been installed) with:
+
+            ```sh
+            brew upgrade litani
+            ```
+
+            ## Ubuntu
+
+            On Ubuntu, install LITANI by downloading the *.deb package below for Ubuntu 20 and install with
+
+            ```sh
+            # Ubuntu 20:
+            $ dpkg -i litani-${{ env.LITANI_VERSION }}.deb
+            ```
+          draft: false
+          prerelease: false
+  ubuntu-20_04-package:
+      runs-on: ubuntu-20.04
+      needs: perform-release
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        LITANI_VERSION: ${{ needs.perform-release.outputs.version }}
+      steps:
+        - uses: actions/checkout@v2
+          with:
+            submodules: recursive
+        - name: Fetch dependencies
+          run: sudo apt-get install -y mandoc scdoc ninja-build
+        - name: make dir structure
+          run: mkdir -p litani-${{ env.LITANI_VERSION }}/{DEBIAN,usr/{bin,libexec/litani,share/{doc/{litani},man/{man1,man5,man7}}}}
+        - name: Create a control file
+          run: |
+            touch litani-${{ env.LITANI_VERSION }}/DEBIAN/control
+            cat << EOF > litani-${{ env.LITANI_VERSION }}/DEBIAN/control
+            Package: Litani
+            Version: ${{ env.LITANI_VERSION }}
+            Architecture: amd64
+            Depends: ninja-build, gnuplot, graphviz, python3.9, python3-jinja2, python3-voluptuous
+            Maintainer: Kareem Khazem <karkhaz@amazon.co.uk>
+            Description: A program that provides platform-independent job control.
+              AWS Build Accumulator collects build jobs from multiple sources before executing them concurrently. It provides platform-independent job control (timeouts, return code control) and an output format that is easy to render into reports (for example, using the built-in renderer). AWS Build Accumulator shines where your project uses multiple different build systems or requires a unified interface describing heterogeneous build jobs.
```

Comment:
Can this line be wrapped at a column width of 80?

Reply:
Resolved